CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 
 - `ExpandSelection` is now a configurable mouse binding action
+- The config watcher now watches the base config paths in addition to symbolic link targets
```

Comment:
I find this phrasing a bit difficult. "Base config path" for me sounds canonicalized, but in this case means the symlink itself. Maybe making it a bit simpler would make things easier to understand.

I'd propose something like this maybe:
```suggestion
- Reload configuration files when their symbolic link is replaced
```

I think that this documentation should be easier to understand, more directly correlated to the changes a user would expect and more concise. The wording "their symbolic link is replaced" is still a bit imprecise and awkward, though it's probably fine.